### ISSUE TITLE
Issue with `mkdocs_autorefs` 0.4.0

### ISSUE THREAD

**Issue by aminalaee** [mkdocstrings/mkdocstrings#398]
**Please open an issue on [pytkdocs](https://github.com/pawamoy/pytkdocs/issues) instead
if this is related to Python docstrings parsing or the collection of Python objects!**

**Describe the bug**
After installing `mkdocstrings` which tries to install `mkdocs_autorefs` there seems to be an issue in `mkdocs_autorefs` latest version 0.4.0 causing module not found.

**To Reproduce**
Installing latest `mkdocstrings` won't work:

requirements:

```
mkdocs==1.2.3
mkdocs-autorefs==0.4.0
mkdocs-material==8.1.11
mkdocs-material-extensions==1.0.3
mkdocstrings==0.18.0
mkdocstrings-python-legacy==0.2.2
```

**Expected behavior**
Working vertsion, maybe by limiting the `mkdocstrings` dependency on `autorefs`.

**Screenshots**

**Information (please complete the following information):**
- Browser: [e.g. chrome, safari]
- `mkdocstrings` version: [e.g. 0.10.2] 

**Additional context**

**Comment by aminalaee** [comment]
I faced this in my pipeline here at SQLAdmin: https://github.com/aminalaee/sqladmin/runs/5449437546?check_suite_focus=true

**Comment by pawamoy** [comment]
Fixed in mkdocs-autorefs 0.4.1, sorry for the CI failures :grinning:

**Comment by aminalaee** [comment]
Thanks for the great work 👍
